light-client-verifier/Cargo.toml

```diff
@@ -35,3 +35,4 @@ flex-error = { version = "0.4.4", default-features = false }
 
 [dev-dependencies]
 tendermint-testgen = { path = "../testgen", default-features = false }
+sha2 = { version = "0.9", default-features = false }
```

Comment:
Is this needed by default, or only when the "rust-crypto" feature is on? Or perhaps no longer needed as a dependency here at all, since the crypto functionality in light-client-verifier should be completely abstracted via `CryptoProvider`.

Reply:
Sorry, it's dev-dependencies, ignore this.